### PR TITLE
disabling console appearing on windows when running in pythonw

### PR DESCRIPTION
**Pull request by Q1Dian**
**Thanks for contributing to Selenium!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description

Hiding console when running in pythonw

### Motivation and Context

When running in pythonw, there will be a console popup that annoys the user (me as i have OCD). I found out that there is a argument STARTUPINFO in windows that makes subprocess.popen not display the console.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x ] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by titusfortner** [comment]
This is not very elegant. I'm not familiar with the constructor can we set properties on it after creating the instance?

**Comment by isaulv** [comment]
@titusfortner This is correct behavior but I wonder if we should make this configurable.

**Comment by titusfortner** [comment]
I think it is correct behavior to not open a separate console on Windows, so I don't think it needs to be configurable. I was hoping the solution would be a little more succinct. Maybe we just split it into 2 different methods for windows/non-windows?

**Comment by isaulv** [comment]
The current method already exists. I'm fine with it really.

**Comment by titusfortner** [comment]
Fair enough. If it looks good, approve the review?

File: py/selenium/webdriver/common/service.py [hunk 1]
```diff
@@ -203,16 +203,32 @@ def _start_process(self, path: str) -> None:
         cmd.extend(self.command_line_args())
```
Comment (captain-hoity-toity) [inline py/selenium/webdriver/common/service.py:203]: I think there's a simpler way to do this.

**Comment by titusfortner** [comment]
Thanks @JefferyVin

**Comment by Q1Dian** [comment]
np, glad to help @titusfortner

**Pull request merged**